### DISCUSSION TITLE
Optimizing Org/Project Structure for Cost Efficiency

### DISCUSSION THREAD

**Discussion by chrishuman0923** [supabase/supabase#28494] [Questions]
Hey everyone, I run a small bespoke software company that develops web-based solutions for my clients. I have a question regarding the ideal structure for organizing Supabase orgs and projects to maximize cost efficiency. I currently pay for Supabase's pro tier for all of my client's production instances (under a single org), but each client also needs a staging instance. These staging instances don't need to run all the time after the production rollout, as some clients won't have frequent changes that need to be reviewed.

To save money, I'd like to be able to pause these staging instances, but this isn't an option on the pro plan. Additionally, I have more than 2 clients, so the free tier isn't a sustainable option unless I create multiple Supabase accounts (one for each client), which I'd like to avoid. I prefer the simplicity of having a single login to manage all my clients' projects on the Supabase dashboard.

At the moment, I'm juggling pausing and unpausing various staging projects on the free tier, but this only works if a maximum of 2 clients request changes at the same time, since I can't have more than 2 free tier projects live at once.

I'm looking for advice or strategies on how to structure my Supabase orgs and projects effectively to minimize costs while accommodating both production and staging needs.

Any insights or recommendations would be greatly appreciated!

Thanks in advance!

**Comment by chrishuman0923** [comment]
Seriously, does no one have an opinion to share on this? Kinda shocked.

**Comment by GaryAustin1** [comment]
Seems like you covered the limits pretty well.  You ruled out each customer having a free account with their credentials, and summed up the inability to pause pro plan instances. 

You can I believe move instances between orgs but your free org is limited to two active instances so pausing would be required.

I don't know how the billing occurs if you transfer a free plan to your pro org for less than a month.  I know add on CPU is prorated but not sure how the base instance is billed.  If partial then you could role things in to the pro plan and back out to the free for pausing.   

Lot of work for $10 x clients for staging.

**Reply by chrishuman0923** [reply]
> I don't know how the billing occurs if you transfer a free plan to your pro org for less than a month.

This is how I initially discovered this. The answer is that you are charged for the usage (prorated) as you correctly surmised. If the testing instance is left on to run for the entire month, it would have been an additional $10.

I am not opposed to passing that additional $10 on to my clients, just wanted to confirm there is no better option. I am just doing my due diligence to make sure I am treating my clients fairly and not charging them unnecessarily because I didn't properly understand my options. 

Alternatively, it looks like Supabase branching might be a solution vs multiple projects but it has the same cost of ~$10 a month if left on. So 🤷🏻‍♂️
